tests/unit/test_sandbox.py

```diff
@@ -7,11 +7,11 @@
 
 
 user_input = """
-from ehrql import Dataset
+from ehrql import create_dataset
```

Comment:
Thanks. We need to do a clean-sweep of the codebase for instances of this at some point because they occur quite frequently in the tests.